src/entities/Client.java

```diff
@@ -0,0 +1,17 @@
+package entities;
+
+public class Client extends User{
+    private Long code;
```

Comment:
What do you think about making code String?

Reply:
Why? I mean, user has a String attribute called identification, that will store the identification card of the client. And also user extends from BaseEntity<Long>

Reply:
Code only will be numbers. If you would like to have more control on the length and the possible combinations of the code you couldn't do it. For example:

1. What if I would like to control client codes by a prefix/suffix indicating their impact in the company with an `A`?
2. Java Long is an 8 bytes data type, making possible combinations of 2^64.
3. You can't control code length in java since 0 is a neutral number if it is the most left number. Which will lead to create extra code for formatting across the whole application.
4. Strings can be truncated at a certain point, while Long has a defined limit.
5. You can increase permutation by 26 possible letters out of numbers.
6. You can add a readable format using strings

However I'm fine if you use Long, but I don't think it will let you be flexible if the company would like to stablish code standards depending on the location or whatever they need

Reply:
Reading that, i think using String can be a good idea, what if i also use it on employee? Maybe an employee can can have a String code that starts with a letter depending the type of employee. But i would have to change User right? because everything (until now) extends from User, i also will have to make some changes there

Reply:
Just use it for client now, maybe later if you need you can just move it

Reply:
done